dotnet/src/Connectors/VectorData.Abstractions/VectorData.Abstractions.csproj

```diff
@@ -30,17 +31,31 @@ Microsoft.Extensions.VectorData.IVectorStoreRecordCollection&lt;TKey, TRecord&gt
     <PackageProjectUrl>https://dot.net/</PackageProjectUrl>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net462' ">
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net462' ">
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
+    <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0' Or '$(TargetFramework)' == 'net462' ">
-    <Compile Include="$(RepoRoot)/dotnet/src/InternalUtilities/src/Diagnostics/IsExternalInit.cs" Link="Diagnostics/IsExternalInit.cs" />
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net462' ">
+    <PackageReference Include="System.ValueTuple" />
```

Comment:
Quoting @mgravell:

> Friends don't let friends use ValueTuple.


In the past, System.ValueTuple has caused a lot of pain related to assembly binding redirects for Full Framework users. I would use it only if we have to.

Reply:
Let's keep it as is and get back to it when we discuss the dependencies of the connector packages.